components/CrossChainInfoModal.tsx

```diff
@@ -53,21 +50,25 @@ const BridgeTransactionModal = ({ isOpen, onClose, txHash }: Props) => {
                   as="h3"
                   className="flex items-center text-base font-semibold leading-6 text-gray-900"
                 >
-                  <InformationCircleIcon className="mr-1 h-5" /> Cross Chain Info
+                  <InformationCircleIcon className="mr-1 h-5" /> Bridge Status Details
                 </Dialog.Title>
                 <div className="flex flex-col items-center">
                   <p className="text-sm py-2 mt-4 w-3/4">
                     We are currently waiting for the relayer to relay your transactions. Your tokens
                     will arrive <strong>~20</strong> minutes after the transaction is confirmed.
+                    Your cross chain transaction is in the queue to be relayed. Your tokens will
```

Comment:
delete the old copy?

Reply:
Fixed during rebase https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/130/commits/f6c4aa8cd7035b2f8400055becdcc01d4ad31b0a